test/src/pub_license/fixtures/generate_pub_license_fixtures.dart

```diff
@@ -0,0 +1,74 @@
+/// A small script used to generate the fixture for the pub_license test.
+///
+/// Fixtures are simply a temporary snapshot of an HTML response from pub.dev.
+/// The generated fixtures allow testing pub_license scraping logic without
+/// making a request to pub.dev every time the test is run.
+///
+/// To run this script, use the following command:
+/// ```bash
+/// dart test/src/pub_license/fixtures/generate_pub_license_fixtures.dart
+/// ```
+///
+/// Or simply use the "Run" CodeLens from VSCode's Dart extension if running
+/// from VSCode.
+library generate_pub_license_fixtures;
+
+// ignore_for_file: avoid_print
```

Comment:
```suggestion
```

Reply:
This is currently being used since I'm printing the progress in lines 54 and 72. I could make the ignores per line or use something like mason logger. What would you like to be done here?

Note that this is intended to be run by developers to quickly generate the fixtures so I'm not too worried about it.